code/game/objects/machinery/cryopod.dm

```diff
@@ -279,10 +279,12 @@
 		return
 
 	user.forceMove(src)
-
 	occupant = user
 	update_icon()
 
+	if(istype(src, /obj/machinery/cryopod/evacuation)) //Will ghost you if it's an evac pod
+		user.ghostize(FALSE)
+
```

Comment:
Why are you istype srcing instead of just return TRUE/FALSE on success/fail and checking that?

Reply:
Where should I be checking it from?